lib/elixir/lib/uri.ex

```diff
@@ -73,6 +73,9 @@ defmodule URI do
   Keys and values can be any term that implements the `String.Chars`
   protocol, except lists which are explicitly forbidden.
 
+  To encode a list use keyword list with multiple mentions of a key
```

Comment:
Ok, thanks. Sorry, I check it one more time and it doesn't work actually: "=" will be encoded into "%3D".